LFS171x/docs/the-promise-of-business-blockchain-technologies.md

```diff
@@ -1 +1,347 @@
+
+The trade finance industry can also leverage information visible in a supply chain blockchain. In its broadest sense, trade finance manages capital required for international trade. Trade financing has become the norm for cross border transactions, with the [World Trade Organization](https://www.wto.org/english/res_e/booksp_e/tradefinsme_e.pdf) estimating that "up to 80 percent of global trade is supported by some sort of financing or credit insurance" (2016). An exporter needs to to mitigate the risk of non-payment, while an importer wants to mitigate the supply risk. The function of trade finance is to act as a third party to remove the payment risk and the supply risk, whilst providing the exporter with accelerated receivables, and the importer with extended credit. Institutions that provide capital during these trades can leverage the information visible in a supply chain blockchain to better evaluate companies for lending.
+
+|![Supply Chain of Tuna](../images/the-promise-of-business-blockchain-technologies/Supply_Chain_of_Tuna.png)|
+|:--:|
+
+<p align="center">
+Source: Nishan Degnarain (used with permission)
+</p>
+
+### Provenance (Part I)
+
+As the previous section on blockchains for supply chain management illustrated, blockchain data improves insight into products, as they move through their lifecycle. Large enterprises are not the only parties to benefit from this increased visibility. Consumers can also benefit from blockchain technology.
+
+Provenance is a record of ownership used as a guide to authenticity or quality. Because of the overhead involved in traditional provenance records, they were only available for very large ticket items, such as works of art. With the efficiencies gained from blockchain technology, provenance records can be available for a wider range of goods. This improved information can aid consumers as they make purchasing decisions.
+
+|![Provenance Use Case](../images/the-promise-of-business-blockchain-technologies/Provenance_Use_Case.jpg)|
+|:--:|
+
+<p align="center">
+How Provenance Works (by [Project Provenance Ltd](https://www.provenance.org/), used with permission)

```

Comment:
For some strange reason, this is not showing up as a link for me when I view it. Instead, it shows up as:
```
How Provenance Works (by [Project Provenance Ltd](https://www.provenance.org/), used with permission)
```

I don't see anything wrong with it.